src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
```

Comment:
Consider using a more descriptive name for 'rentDay'. For example, 'DAILY_RENTAL_RATE' would provide clearer information about what this constant represents.